test/unit/snowflake_test.js

```diff
@@ -816,9 +816,6 @@ describe('connection.execute() statement failure', function () {
                 assert.strictEqual(statement.getNumRows(), undefined);
                 assert.strictEqual(statement.getSessionState(), undefined);
 
-                assert.strictEqual(statement.getStatementId(), null);
```

Comment:
Should we verify the value here? Or at least - is not null?

Reply:
I added checks for value being not null string.